mne/externals/FieldTrip.py

```diff
@@ -309,7 +309,7 @@ def getHeader(self):
                 (chunk_type, chunk_len) = struct.unpack(
                     'II', payload[offset:offset + 8])
                 offset += 8
-                if offset + chunk_len < bufsize:
+                if offset + chunk_len > bufsize:

```

Comment:
okay I was trying to read the code and understand but I think I'll give up on that for now. Tests pass so I think we should be fine. Let's merge @Eric89GXL !

Here's my neuromag2ft++ ;) http://perso.telecom-paristech.fr/~mjas/neuromag2ft-3.0.2.zip. It can also do some funky stuff like do online MaxFilter etc. You can also specify a speedup factor when simulating from files.